bin/src/run/index.ts

```diff
@@ -19,8 +20,23 @@ export default function runRollup(command: any) {
 		}
 	}
 
-	if (command.output && command.output.dir) {
-		command.input = command._;
+	if (command.dir) {
```

Comment:
One thing I noticed not directly related to this but which we might fix here as well: Apart from directly providing multiple file names on command line, people might also just want to use the `--input` option multiple times. This is actually supported by our command line parser which will turn the input option into an array in this case but rollup will not handle this correctly. It should be easy to transform this case properly here as well.

Reply:
It seems odd that we support both `rollup x.js` and `rollup --input x.js` as meaning the same thing... perhaps we should even consider deprecating the `--input` option then, as `rollup x.js y.js z.js --dir out` seems preferable to `rollup --input x.js --input y.js --input z.js --dir out`?

Reply:
Yes and no. When there are many options and several inputs, it can help readability if you can precede each input with `--input`.

Reply:
Sure, I've added support for this.